spec/p2p/v0.34/transport.md

```diff
@@ -0,0 +1,223 @@
+# Transport
+
+The transport establishes secure and authenticated connections with peers.
```

Comment:
I had a similar question before: Is transport a reactor? Or a package, a protocol is more appropriate way to conceptualize it? I would find it more easy to understand it if we qualify it by some type/label.

Reply:
My understanding of reactors is that each one is a major component in Tendermint (consensus, mempool, blocksync, statesync, p2p etc.). So transport itself would probably not be where the p2p reactor is implemented. However, I agree that clarifying it's scope and defining it as either a package or protocol would help.